lib/module-resolvers/file-resolver.js

```diff
@@ -39,22 +39,58 @@ class FileResolver {
         }
     }
 
-    _search(module_path, module_name, importer_path) {
+    _canonicalize(filename) {
+        if (path.extname(filename) !== '.juttle') {
+            filename = filename + '.juttle';
+        }
+
+        return filename;
+    }
+
+    _search_local_import(module_path, module_name, importer_path) {
         var self = this;
 
-        logger.debug(`Resolving module_path=${module_path} module_name=${module_name} importer_path=${importer_path}`);
+        logger.debug(`searching local module_path=${module_path} module_name=${module_name} importer_path=${importer_path}`);
+
+        // Resolve module_path relative to importer_path or the
+        // current working directory.
+        let dir = process.cwd();
+        if (importer_path !== undefined &&
+            path.isAbsolute(importer_path)) {
+            dir = path.dirname(importer_path);
+        }
```

Comment:
Would be good to add a comment on when the importer path is non-absolute (basically your [answer here](https://github.com/juttle/juttle/pull/591#discussion_r56397960)).